ring_doorbell/auth.py

```diff
@@ -47,44 +83,95 @@ def __fetch_token(self, username, password,
         :type auth_code: str
         """
         if auth_code:
-            headers = {}
+            headers = OAuth.HEADERS.copy()
```

Comment:
Isn't the only difference in this if-statement the headers ? We could clean it up like this:

```python
if auth_code:
    … # Existing
else:
    headers = OAuth.HEADERS
```